<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderManager</class>
 <widget class="QDialog" name="OrderManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>订单管理</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>780</width>
     <height>580</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <!-- 搜索栏 -->
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="leOrderSearch">
        <property name="placeholderText">
         <string>输入订单号、客户姓名或房间号搜索...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    
    <!-- 筛选条件和订单列表 -->
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <!-- 筛选条件 -->
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>筛选条件</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <!-- 日期范围 -->
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>入住日期范围:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QDateEdit" name="deStartDate">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>至</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="deEndDate">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         
         <!-- 订单状态 -->
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>订单状态:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbOrderStatus"/>
         </item>
         
         <!-- 房型 -->
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>房型:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbRoomType"/>
         </item>
         
         <!-- 操作按钮 -->
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="btnResetFilter">
             <property name="text">
              <string>重置筛选</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnClose">
             <property name="text">
              <string>关闭</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      
      <!-- 订单列表 -->
      <item>
       <widget class="QTableView" name="tableView">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
